src/ReleaseHistory.md

```diff
@@ -1,5 +1,8 @@
 # SARIF Website Release History
 
+## **v1.0.1**
+- Updating contributors list/images, bump Sarif.Multitool.Library from 2.4.14 to 2.4.16, add link to java-sarif documentation. [#142](https://github.com/microsoft/sarif-website/pull/142)
```

Comment:
>contributors [](http://example.com/codeflow?start=11&length=12)

Just curious, didn't we hear from Contrast Security that they'd be willing to be badged on the site?

Reply:
Possibly, but to my knowledge they have not submitted the graphic for inclusion at this time. @EasyRhinoMSFT, have you heard from them at all?